### PR TITLE
Added a bunch of splashes

### PR DESCRIPTION
**Pull request by ashy1227**
#### What is this PR for?

I added a bunch of splash texts...I had an idea for one and well that turned into 50 lol

Some pony quotes, Minecraft splash parodies, quotes from FiW, song references galore, and some miscellaneous pony stuff, just check the preview :P feel free to remove any if u don't like

- [X] **I have made and am willing to license my contribution under the MIT license, or have permission from the original author to license it under MIT.**

### CODE REVIEW

File: assets/minecraft/texts/splashes.txt [hunk 1]
```diff
@@ -312,3 +312,53 @@ Step 2. ????
...
+Just remembered what happened in September
+Friendship is power!
+The antonymph of the internet!
+Still cleaning up the viruses that you have left!
+Don't stop until you hear the §e§oyay§r§e!
+Element of harmony inside!
+Sweet Celestia!
+Don't feed the parasprites!
+Bungled all its bangles!
+Rife with devastation!
```
Comment (meadowsys) [inline assets/minecraft/texts/splashes.txt:334]: hmm, i'm not sure i know what this means? (Rife with devastation!)

File: assets/minecraft/texts/splashes.txt [hunk 2]
```diff
@@ -312,3 +312,53 @@ Step 2. ????
...
+Chortle at the cooky!
+Snortle at the spooky!
+Laugh and make them disappear!
+Is there any way to fix a broken heart?
+You've got the music in you!
+In a city not some back water!
+Cider! Cider! Cider! Cider!
+Puns beyond belief!
+The horror! The horror!
+More like An§e§oKony§r§e C, amirite?
```
Comment (meadowsys) [inline assets/minecraft/texts/splashes.txt:357]: this one too.. >~< (More like AnKony C, amirite?)

**Review by meadowsys — CHANGES_REQUESTED** [review]

File: assets/minecraft/texts/splashes.txt (hunk 1 above)
Comment (ashy1227) [reply assets/minecraft/texts/splashes.txt:334]: From the gypsy bard song! :b maybe a different verse would be better "trapped inside a crystal ball" "a toy maker's creation"

File: assets/minecraft/texts/splashes.txt (hunk 2 above)
Comment (ashy1227) [reply assets/minecraft/texts/splashes.txt:357]: That's a reference to this video by Antony C <https://youtu.be/BsIa_LKojJI>

File: assets/minecraft/texts/splashes.txt (hunk 1 above)
Comment (meadowsys) [reply assets/minecraft/texts/splashes.txt:334]: ahhh, okie!

File: assets/minecraft/texts/splashes.txt (hunk 2 above)
Comment (meadowsys) [reply assets/minecraft/texts/splashes.txt:357]: hmm, oke

**Comment by meadowsys** [comment]
hmm, i'm going to say everything looks good!

**Review by meadowsys — APPROVED** [review]

**Pull request merged**